xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -71,6 +71,9 @@ bool CPVRChannelGroups::Update(const CPVRChannelGroup &group, bool bUpdateFromCl
     if (group.GroupID() > 0)
       updateGroup = GetById(group.GroupID());
 
+    if (!updateGroup && group.IsInternalGroup() && (updateGroup = GetGroupAll()) && bUpdateFromClient)
```

Comment:
I said I won't bitch about it this time, have bitched enough for this PR ;-) But yes, a method starts at the top and ends at the last bracket, returning something that you've changed in the middle, returns and continues are evil ;-)

As for this change, I was already wondering what the rationale for this line was, which is why I asked for a comment.